packages/core/helper-plugin/src/utils/stopPropagation.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+export const stopPropagation = {
```

Comment:
Sure I'll be happy to do it, but do you know why we want to get rid of it, and what we recommend instead?

Reply:
I think because it does more than what it says (adds role and aria-hidden). Personally I don't really see the need for this util. I would just recommend using these properties directly where needed as needed. What do you think?

Reply:
I second what @markkaylor says, it implicitly makes things `button` which can be _totally_ wrong 👍🏼